doc/release-notes/11252-ncml.md

```diff
@@ -0,0 +1,18 @@
+## NcML Previewer Bug Fix
+
+[Dataverse Previewers](https://github.com/gdcc/dataverse-previewers) v1.4 contains a bug in the NcML previewer that prevents it from working. (See #11252 for screenshots.)
```

Comment:
It only doesn't work with signedUrls. Should still be fine with an API key.

Reply:
Thanks, fixed in bce2d31285